deepface/detectors/DetectorWrapper.py

```diff
@@ -140,7 +155,9 @@ def detect_faces(
 
 
 def rotate_facial_area(
-    facial_area: Tuple[int, int, int, int], angle: float, direction: int, size: Tuple[int, int]
+    facial_area: Tuple[int, int, int, int],
+    angle: float,               # in degrees. The sign determines the direction of rotation
```

Comment:
we should not have comment in this line, this is already explained in its docstring